specification/_global/get/GetResponse.ts

```diff
@@ -18,7 +18,14 @@
  */
 
 import { GetResult } from '@global/get/types'
+import { ErrorResponseBase } from '@_types/Base'
 
 export class Response<TDocument> {
   body: GetResult<TDocument>
+  exceptions: [
+    {
+      statusCodes: [404]
+      body: GetResult<TDocument> | ErrorResponseBase
```

Comment:
Can you add a comment explaining that `ErrorResponseBase` will be returned if the index doesn't exist, and `GetResult` if the id doesn't exist, and in that case it will only contain "found": false, _id and _index? 

And for the sake of nitpicking flip the two members of the union: start with the result for the unknown index and finish with the result for the unknown id 😉

Reply:
Documentation added!

The order is consistent with the existing exceptions in the rest of the specification; I suggest we keep it that way.